src/map/clif.c

```diff
@@ -481,16 +481,16 @@ static bool clif_send(const void *buf, int len, struct block_list *bl, enum send
 			/* Fall through */
 		case AREA_WOC:
 		case AREA_WOS:
+			Assert_retr(false, bl->m >= -1 && bl->m < map->count);
```

Comment:
Better add an exception for floatings NPCs and check `bl->m >= 0` for non-NPC objects.
`Assert_retr(false, ((bl->type == BL_NPC && bl->m == -1) || bl->m >= 0) && bl->m < map->count);`

Same in `case AREA_CHAT_WOC:`.

Reply:
map check in clif probably useless. i add it here for early catch possible bugs. but all this checks useless for current bug, because ``m`` always valid